static/src/js/actions/__tests__/errors.test.js

```diff
@@ -79,3 +84,36 @@ describe('removeError', () => {
     expect(removeError(payload)).toEqual(expectedAction)
   })
 })
+
+describe('handleError', () => {
+  test('should dispatch error with showAlertButton when showAlertButton parameter is true', async () => {
+    const consoleMock = jest.spyOn(console, 'error').mockImplementation(() => jest.fn())
```

Comment:
You still aren't doing anything with this console mock. If you mock something it is normally because you expect it to be called, so you need to do assertions for it.

You also shouldn't need `consoleMock.mockRestore()`, the global mock cleanup should handle that